_docs/setup/install-kubernetes.md

```diff
@@ -36,21 +37,18 @@ Quick Start instructions to install and configure Istio in a Kubernetes cluster.
  oc adm policy add-scc-to-user anyuid -z istio-ingress-service-account -n istio-system
  oc adm policy add-scc-to-user anyuid -z istio-egress-service-account -n istio-system
   ```  
-You can expose services either via istio ingress or using default openshift router.
-  
+
 * Install the Kubernetes client [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/), or upgrade to the latest
```

Comment:
can we also dictate we need >1.7.0 kubectl for the CRD stuff?

Reply:
it's already on line 18 above

Reply:
no, that is kube cluster version.  this is kubectl version.

Reply:
well clearly kubectl and cluster versions should match, seems obvious to me but ok
if we want to be super detailed it should be 1.7.4 not 1.7.0 etc. etc... I think those very specific details are better for release notes though (I would put kubernetes 1.7 and that's it)

Reply:
i think  v1.7 is fine in the doc for server and client. people don't always move client and server together as it is not required. however, given the prior v1.7 client doesn't support crd, we should be clear that we need that level of client.

Reply:
tell me if you think #479 is clearer/helps